charts/datarepo-api/README.md

```diff
@@ -92,3 +92,5 @@ The following tables lists the configurable parameters of the Datarepo-api chart
 | `existingApplicationSecretSecretKey`                        |The key in the existing secret that stores the Azure application secret                                                                                                                                               | `nil`                                                         |
 | `existingSecretRBS`                        |Name of an existing secret that stores the RBS service account credentials                                                                                                                                          | `nil`                                                         |
 | `existingRBSSecretKey`                        |The key in the existing secret that stores the RBS service account credentials                                                                                                 | `nil`                                                         |
+| `existingSynapseUserSecretKey`                        |The key in the existing secret that stores the azure synapse sql admin user name                                                                                                 | `nil`                                                         |
```

Comment:
I think you'll want 
`secretsgeneric.synapseUserSecret` and `secretsgeneric.synapsePasswordSecret` starting on line 88

Reply:
I think it's for backwards compat. (I think for fake prod).  Note that you'd also need to add an entry to data in
charts/datarepo-api/templates/secrets.yaml

and

charts/datarepo-api/templates/values.yaml

Reply:
Oh, ok! It doesn't look like we've done with for every secret. Do you know why it's only sometimes needed?

Reply:
Should be added now